src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArr = people.filter(person => {
+    return !century
+      ? person.sex === 'm'
+      : person.sex === 'm' && century === Math.ceil(person['died'] / 100);
+  });
+  const arrAge = menArr.map(year => year.died - year.born);
+  const result = arrAge.reduce((manA, manB) => manA + manB);
```

Comment:
this is not result
the first argument of the reduce method is not manA, that is accumulator